source/manual/upgrade-rails.html.md

```diff
@@ -0,0 +1,28 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-10-28
```

Comment:
This needs to be changed after https://github.com/alphagov/govuk-developer-docs/pull/185.

```
last_reviewed_at: 2017-04-28
review_in: 6 months
```